.prow.yaml

```
@@ -0,0 +1,15 @@
+presubmits:
+  - name: pull-controller-runtime-everything
+    always_run: true
+    decorate: true
+    clone_uri: "ssh://git@github.com/kcp-dev/controller-runtime.git"
+    labels:
+      preset-goproxy: "true"
+    spec:
+      containers:
+        # TODO(MadhavJivrajani): Change the following to the kcp-dev/build image once it is available
+        # Ref: https://github.com/kcp-dev/infra/pull/34
+        - image: quay.io/kubermatic/build:go-1.20-node-18-6
```

https://github.com/kcp-dev/infra/pkgs/container/infra%2Fbuild is available now, so I would suggest switching to that. If we are missing tools from that, we can add them as needed.

```suggestion
        - image: ghcr.io/kcp-dev/infra/build:1.19.9-2
```